sass/components/_popover.scss

```
@@ -75,6 +75,7 @@
 
             &:after {
                 border-color: transparent;
+                border-bottom-color: v(center-channel-bg);
```

Sorry! I believe I approved a little pre-maturely. Are we able to change the `v` to `var` and add `--` to the front of the variable. Thanks. (This is copying the pattern done elsewhere. For example https://github.com/mattermost/mattermost-webapp/blob/master/components/generic_modal.scss#L106 )
```suggestion
                border-bottom-color: var(--center-channel-bg);
```

@hmhealey or @deanwhillier 
Can you guys please confirm the requested change is corrected. This is after reviewing the sample PR. I just want to double check or if I should approve this right away. Thank you

---

Nice catch @hahmadia I guess I also was in a spur of approvals

---

I will make this change real quick! Should I also change this thing in other PRs?